components/src/dynamo/frontend/sglang_prepost.py

```
@@ -307,26 +501,93 @@ def process_output(self, engine_response: dict[str, Any]) -> dict[str, Any] | No
+                    if (
+                        not final_calls
+                        and self._tool_call_parser_name
+                        and self._sglang_tools
+                    ):
+                        try:
+                            fcp = FunctionCallParser(
+                                tools=self._sglang_tools,
+                                tool_call_parser=self._tool_call_parser_name,
+                            )
+                            _, final_calls = fcp.parse_non_stream(full_text)
+                        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)